src/integrations/VWO/browser.js

```diff
@@ -122,6 +122,8 @@ class VWO {
               this.analytics.track('Experiment Viewed', {
                 experimentId: expId,
                 variationName: _vwo_exp[expId].comb_n[variationId],
+                CampaignName: _vwo_exp[expId].name,
```

Comment:
It is coming as `undefined` in sonarcloud analysis. Any issues?

Reply:
_vwo_exp will be defined when vwo SDK will be loaded.